src/store/authentication/types.ts

```diff
@@ -37,3 +37,13 @@ export interface User {
 export interface AuthenticationState {
   user: UserPayload;
 }
+
+export interface Member {
```

Comment:
let's put this in either the channel store or the user. if we're using the terminology of 'user' globally in the app, then let's rename this, and keep it with the user interface. or else just keep it scoped to the channel if we just want it specifically to be for channels, in which case it can be called Member, if we're calling channel member 'Members'